### PR TITLE
Remove uses of "implicit" that now refer to a default qualifier.

### PR DESCRIPTION
**Pull request by smillst**
Remove uses of the word "implicit" that refer to the old notion of "an implicit qualifier" which now should just be a "default qualifier".

### CODE REVIEW

**Comment by mernst** [comment]
@smillst The title is 
> Remove uses of "implicit" that refer to an implicit qualifier.

 Should there be a negation, or a different word, in there?

**Comment by smillst** [comment]
I mean:

Remove uses of the word "implicit" that refer to the old notion of "an implicit qualifier" which now should just be a "default qualifier".  

Does that make sense?

File: framework/src/main/java/org/checkerframework/framework/qual/LiteralKind.java [hunk 1]
```diff
@@ -11,8 +11,7 @@
  * that enum is in the tools.jar which is not on the user's classpath by default. So this enum is
  * used instead.
  *
- * @checker_framework.manual #creating-type-introduction Declaratively specifying implicit
- *     annotations
+ * @checker_framework.manual #creating-type-introduction Declaratively specifying annotations
```
Comment (mernst) [inline framework/src/main/java/org/checkerframework/framework/qual/LiteralKind.java]: Should this mention defaults?  The related section is about type introduction, not about specifying annotations per se.

File: framework/src/main/java/org/checkerframework/framework/qual/UpperBoundFor.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.ANNOTATION_TYPE)
 public @interface UpperBoundFor {
-    /** @return {@link TypeKind}s of types for which an annotation should be implicitly added */
+    /** @return {@link TypeKind}s of types for which an annotation should be added */
```
Comment (mernst) [inline framework/src/main/java/org/checkerframework/framework/qual/UpperBoundFor.java]: "Added by default?"

**Pull request merged**